src/scripts/towns/Town.ts

```diff
@@ -1845,11 +1845,11 @@ TownList['Reflection Cave'] = new DungeonTown(
     [new RouteKillRequirement(10, GameConstants.Region.kalos, 11)]
 );
 //Tower of Mastery?
-TownList['Azure bay'] = new DungeonTown(
-    'Azure bay',
-    GameConstants.Region.kalos,
-    [new RouteKillRequirement(10, GameConstants.Region.kalos, 12)]
-);
+// TownList['Azure bay'] = new DungeonTown(
+//     'Azure bay',
+//     GameConstants.Region.kalos,
+//     [new RouteKillRequirement(10, GameConstants.Region.kalos, 12)]
+// );
```

Comment:
We can just completely remove this, don't think it will ever be needed to bring it back, as it is now a route.
```suggestion
```